packages/terra-paginator/tests/wdio/paginator-spec.js

```diff
@@ -69,4 +69,42 @@ Terra.describeViewports('Controlled Paginator', ['large'], () => {
 
     Terra.it.matchesScreenshot('3');
   });
+
+  describe('Controlled Progressive Paginator without total count', () => {
+    before(() => browser.url('/#/raw/tests/terra-paginator/paginator/controlled-progressive-paginator-without-total-count'));
+
+    Terra.it.validatesElement('0');
```

Comment:
What does 0 represent?

Reply:
Can we update these to use be in their own it blocks since Terra.it helpers are being deprecated?

Reply:
`What does 0 represent?` 
I think it was added to distinguish between the screenshots that were taken for different page numbers in controlled-paginator-test.

Reply:
updated [here](https://github.com/cerner/terra-core/pull/3190/commits/d0e5ef41693dba34291910c0e38aeb195fc36a43#diff-65b614af80147a518e5be5f9679cb2d513196d5503a8cbb0a71f052cbbe10059)